DEVELOPMENT.md

```diff
@@ -1,3 +1,13 @@
+# Version Support Guidelines
+
+Elixir itself supports 5 versions with security updates:
+https://hexdocs.pm/elixir/compatibility-and-deprecations.html#content
+
+OTP Supports the last 3 versions:
+http://erlang.2086793.n4.nabble.com/OTP-Versions-and-Maint-Branches-td4722416.html
+
+ElixirLS supports the last 4 versions of Elixir and the last 3 versions of OTP. However this is not a hard and fast rule and may change in the future.
```

Comment:
IMO we could aim for 3 last versions

Reply:
Okay, after thinking about it some more I agree. I think this will help us focus more on stability (by reducing the number of supported configurations)